module/app/app.go

```diff
@@ -217,30 +217,34 @@ type Gravity struct {
 }
 
 // ValidateMembers checks for nil members
-func (g Gravity) ValidateMembers() {
-	if g.legacyAmino == nil { panic("Nil legacyAmino!") }
+func (app Gravity) ValidateMembers() {
```

Comment:
Go doesn't like it when you have differently named receiver variables, changed g -> app